core/src/script/CGXP/plugins/WFSPermalink.js

```diff
@@ -227,6 +236,10 @@ cgxp.plugins.WFSPermalink = Ext.extend(gxp.plugins.Tool, {
 
         var layerName = state.layer;
         delete state.layer;
+        if (state.show) {
+            this.showFeatures = false;
```

Comment:
Thanks @kalbermattenm this time I understand :-) than the latest purpose looks good :-)